ts/tests/FormBuilderTest.ts

```diff
@@ -270,10 +270,10 @@ describe('Form builder', () => {
             referenceNumber
         );
 
```

Comment:
The usage of "showPaymentMethodSelector" could indicate it's actually deprecated, e.g. change the name and perhaps use the value undefined?